scipy/sparse/_bsr.py

```diff
@@ -94,8 +94,10 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False, blocksize=None):
                     if not isshape(blocksize):
                         raise ValueError(f'invalid blocksize={blocksize}')
                     if tuple(blocksize) != self.data.shape[1:]:
-                        raise ValueError('mismatching blocksize={} vs {}'.format(
-                            blocksize, self.data.shape[1:]))
+                        raise ValueError(
+                            f'mismatching blocksize={blocksize} \
+                                vs {self.data.shape[1:]}'
```

Comment:
Similarly, the `\` at the end of the line means all the indentation spacing is include. Use a single line, or two strings one on each line
```python
                            f'mismatching blocksize={blocksize} vs {self.data.shape[1:]}'
```
or 
```python
                            f'mismatching blocksize={blocksize} '
                            f'vs {self.data.shape[1:]}'
```
(I suggest the single line in this case -- it is short enough.